### PR TITLE
Add redesign work for some user settings pages

### PR DESCRIPTION
**Pull request by eseliger**
Aligns headers and adds containers to the three pages below:

image
image
image

### CODE REVIEW

**Comment by rrhyne** [comment]
Can we wrap research in a container as so:

https://www.figma.com/file/73unFUigMIcWdH2QXidsbm/batch-changes-redesign-issues?node-id=46%3A2802

Small adjustments to the subscription page would also help:

https://www.figma.com/file/73unFUigMIcWdH2QXidsbm/batch-changes-redesign-issues?node-id=46%3A2835

**Comment by rrhyne** [comment]
One more... can we adjust the site admin banner that is scrunched in the left column? Figma:

https://www.figma.com/file/73unFUigMIcWdH2QXidsbm/batch-changes-redesign-issues?node-id=46%3A2923

**Comment by quinnkeast** [comment]
+1 on Rob's comments, and good to go then.

**Review by limitedmage — COMMENTED** [review]
We should probably add storybook tests for the modified components here.

**Comment by rrhyne** [comment]
@limitedmage Erik is donating some batcher time to get this in before the release. If it's ok with you, I'd rather not tap that time further at the moment.

**Comment by limitedmage** [comment]
> @limitedmage Erik is donating some batcher time to get this in before the release. If it's ok with you, I'd rather not tap that time further at the moment.

That works :) We can add stories later.

File: client/web/src/enterprise/user/settings/auth/UserSettingsPermissionsPage.tsx [hunk 1]
```diff
@@ -29,7 +31,7 @@ export const UserSettingsPermissionsPage: React.FunctionComponent<{ user: UserAr
     return (
         <div className="w-100">
             <PageTitle title="Permissions" />
-            <h2>Permissions</h2>
+            <PageHeader headingElement="h2" path={[{ text: 'Permissions' }]} className="mb-3" />
```
Comment (pdubroy) [inline client/web/src/enterprise/user/settings/auth/UserSettingsPermissionsPage.tsx:34]: Not for this PR, but seems like we should clean up this API to make it a bit more convenient to use this for text.

**Review by pdubroy — APPROVED** [review]
lgtm. Thanks!

**Comment by eseliger** [comment]
TODO: change wording on subscriptions page to
"Purchase a subscription for a self-hosted Sourcegraph instance. See <a>pricing</a> for more information."

**Review by valerybugakov — APPROVED** [review]

**Pull request merged**